package.json

```diff
@@ -46,12 +46,16 @@
     "prerelease": "semantic-release --dry-run",
     "dependencies:check": "ncu",
     "dynamodb": "sls dynamodb start",
-    "tools-setup": ""
+    "tools-setup": "sls dynamodb migrate && BRANCH=local gulp start-serverless && sls dynamodb seed --seed=trailer-registration"
```

Comment:
will fail and has to be deleted as it won't be used, it return something as the pipeline requires it

Reply:
Already rebased. Need to push.